RubberduckTests/TodoExplorer/TodoExplorerTests.cs

```diff
@@ -219,5 +226,12 @@ private Configuration GetTodoConfig(string[] markers)
             var userSettings = new UserSettings(null, null, null, todoSettings, null, null, null, null);
             return new Configuration(userSettings);
         }
+
+        private IUiDispatcher GetMockedUiDispatcher()
+        {
+            var dispatcher = new Mock<IUiDispatcher>();
+            dispatcher.Setup(m => m.Invoke(It.IsAny<Action>())).Callback((Action argument) => argument.Invoke());
+            return dispatcher.Object;
+        }
```

Comment:
This mock setup will definitely be used in other tests. Do we not have a central place for this yet?

Reply:
Not yet.  I think? it's currently used by two other test suites (the Code Explorer and unit testing). In those cases, the setup is basically just a one liner in the constructor of the test setup object though.